packages/docs/src/styles/ecosystem-diagram.css

```
@@ -0,0 +1,543 @@
+   in a single tall column, and both gutter arrows point into it - the same
+   left-to-right reading order as the original diagram. Keeping the six library
+   rows beside the left column instead of below it is what makes the figure wide
+   rather than tall.
+   ============================================================================= */
+
+@container eco (min-width: 700px) {
+  .eco-grid {
+    --eco-gutter: 78px;
+    grid-template-columns:
+      minmax(0, 0.92fr) var(--eco-gutter)
+      minmax(0, 1.13fr);
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)